Failure on x86_64-darwin [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34674&attempt_id=f52572fc-9b72-40c4-89aa-aa07dd15a62b)

<details><summary>Partial log (click to expand)</summary><p>

```
build flags: SHELL=/nix/store/s8mff1kmnc63b21ybdid2ni0fw7mzy7r-bash-4.4-p12/bin/bash
  CXX      diff_pdf-diff-pdf.o
  CXX      diff_pdf-bmpviewer.o
  CXX      diff_pdf-gutter.o
  CXXLD    diff-pdf
ld: framework not found Cocoa
clang-4.0: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Makefile:413: diff-pdf] Error 1
builder for '/nix/store/bh68sz2lr6yn0hkzq386jh8biayj07az-diff-pdf-2017-12-30.drv' failed with exit code 2
error: build of '/nix/store/bh68sz2lr6yn0hkzq386jh8biayj07az-diff-pdf-2017-12-30.drv' failed
```
</p></details>

--- a/pkgs/applications/misc/diff-pdf/default.nix
+++ b/pkgs/applications/misc/diff-pdf/default.nix
@@ -1,5 +1,12 @@
-{ stdenv, fetchFromGitHub, autoconf, automake, pkgconfig, cairo, poppler, wxGTK }:
+{ stdenv, fetchFromGitHub, autoconf, automake, pkgconfig, cairo, poppler, wxGTK ? null, wxmac ? null, darwin ? null }:
 
+let
+  wxInputs =
+    if stdenv.isDarwin then
+      [ wxmac darwin.apple_sdk.frameworks.Cocoa ]
+    else
+      [ wxGTK ];
+in
 stdenv.mkDerivation rec {
   name = "diff-pdf-${version}";
   version = "2017-12-30";
@@ -12,7 +19,7 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ autoconf automake pkgconfig ];
-  buildInputs = [ cairo poppler wxGTK ];
+  buildInputs = [ cairo poppler ] ++ wxInputs;
 
   preConfigure = "./bootstrap";
 

